oat-sa/tao-core
# feat: create atomic roles for item class

https://oat-sa.atlassian.net/browse/ADF-154

- Created atomic roles for item classes (ItemClassNavigator, ItemClassEditor, ItemClassCreator) which include base BackOffice for being able to login
- Migration and install script
- Non-breaking fix in RdfController for proper division of access to view and edit methods In child controllers

How to test:

- Create users with atomic roles
- Log in and check that only specified in the task abilities are matched

Dependent PRs:

- [generis](https://github.com/oat-sa/generis/pull/896)
- [taoItems](https://github.com/oat-sa/extension-tao-item/pull/462)
